src/Generator/GiftCardPdfGenerator.php

```diff
@@ -33,4 +39,21 @@ public function generatePdfResponse(
 
         return new PdfResponse($this->snappy->getOutputFromHtml($html), 'gift_card.pdf');
     }
+
+    public function generateAndSavePdf(
```

Comment:
Why isn't this method returning anything? :)

Reply:
It is just generating the PDF and creating the file. So nothing to return here imo.

Reply:
But wouldn't you expect from the name of the method that you either give the filename as input or it returns the filename as output?

Reply:
Well, Since the file path generation is handled in another service, I'm not sure it is necessary. But we can return the name indeed.